src/include/uapi/abi.h

```diff
@@ -52,7 +52,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 2
-#define SOF_ABI_MINOR 0
+#define SOF_ABI_MINOR 1
 #define SOF_ABI_PATCH 0
```

Comment:
I'll let Liam comment on this one, we might just add this without a bump since it has no effect?